### PR TITLE
Add 8.2.0

### PR DESCRIPTION
**Pull request by mzch**
Add 8.2.0. I've confirmed compiling and running `8.2.0` successfully with [Leantime](https://github.com/Leantime/leantime).

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Can you resolve the merge conflicts, please?

File: share/php-build/definitions/8.2.0 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+configure_option "--enable-gd"
+configure_option "--with-jpeg"
+configure_option "--with-zip"
+configure_option "--with-mhash"
+
+configure_option -D "--with-xmlrpc"
+
+install_package "https://www.php.net/distributions/php-8.2.0.tar.bz2"
+install_xdebug "3.2.0RC2"
```
Comment (GrahamCampbell) [inline share/php-build/definitions/8.2.0:9]: 3.2.0 has now been released. please use that. ;)

File: run-tests.sh [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 # Update this when a new stable version comes around
 STABLE_DEFINITIONS="5.3.29 5.4.45 5.5.38 5.6.40 7.0.33 7.1.33 7.2.34 7.3.33 7.4.33 8.0.25 8.1.12"
+STABLE_DEFINITIONS=$STABLE_DEFINITIONS" 8.2.0"
```
Comment (GrahamCampbell) [inline run-tests.sh:5]: plz add this directly into line 4

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: share/php-build/definitions/8.2.0 (hunk 1 above)
Comment (mzch) [reply share/php-build/definitions/8.2.0:9]: OK.

**Comment by mzch** [comment]
Oh, this PR became ugly. I'd like to re-create a new PR, so I just closed this.

**Pull request closed without merging**